src/js/views/patients/shared/bulk-edit/bulk-edit_views.js

```diff
@@ -312,6 +314,11 @@ const BulkEditActionsBodyView = View.extend({
 
     this.showChildView('duration', durationComponent);
   },
+  showApplyOwner() {
+    this.showChildView('applyOwner', new ApplyOwnerView({
```

Comment:
This is the action apply owner so right here I think it can be
```
model: this.model,
isForFlows: false,
```
And then in the other view for flows we can do the opposite and not have to set it on the state model

Reply:
Just updated it to this